### PR TITLE
Blog post about POWER8 FTP servers

### PR DESCRIPTION
**Pull request by ramereth**

### CODE REVIEW

File: content/osuosl/leveling-up-with-power8.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+:status: draft
+
+Over the past year I've had the privilege of working on a new POWER architecture
+ppc64le (PowerPC 64bit Little Endian). We've had a long relationship with IBM at
+the Open Source Lab (OSL) providing resources for the POWER architecture to FOSS
+projects.
+
+Earlier this year, IBM graciously donated three powerful POWER8 machines to the
+OSL to replace our aging FTP cluster. This produced a few challenges we needed
+to overcome to make this work:
```
Comment (jordane) [inline content/osuosl/leveling-up-with-power8.rst]: Did you want to mention the work with LE vs BE and Opal and RH 7.1 insanity? I guess that could be perceived negatively though.

Comment (ramereth) [inline content/osuosl/leveling-up-with-power8.rst]: The licensing stuff? No I didn't want to bring that up.

Comment (jordane) [inline content/osuosl/leveling-up-with-power8.rst]: I meant more just figuring out what to run and using Opal over HMC, but yeah.

**Comment by jordane** [comment]
@ramereth it might be worth mentioning our choice of using the hardware raid over sw raid + bcache.

File: content/osuosl/leveling-up-with-power8.rst [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+proprietary management system called the Hardware Management Console (HMC). It
+was an extremely difficult system to use and was so foreign to how we normally
+manage systems. So when we got our first POWER8 system, I was delighted to see
+that they did away with the HMC and provided an abstraction layer called OPAL to
+manage the system. Basically, it meant these machines actually use open
+standards and essentially boot like an x86 machine more or less (i.e. what we're
+more used to).
+
+When you first boot a POWER8 machine that is using OPAL, you use IPMI to connect
+to the serial console (which needs to be enabled in the FSP). When it first
```
Comment (jordane) [inline content/osuosl/leveling-up-with-power8.rst]: I'd recommend expanding FSP.

File: content/osuosl/leveling-up-with-power8.rst [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+boots up, you'll see a Linux kernel booting up and then a boot prompt running
+Petitboot. Petitboot is an kexec based boot loader.
```
Comment (jordane) [inline content/osuosl/leveling-up-with-power8.rst]: This should be `is a kexec based ...`

File: content/osuosl/leveling-up-with-power8.rst [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+This basically allows you to do the following:
+
+* Auto-boots a kernel
+* Gives you a bash prompt to diagnose the machine or setup hardware RAID
+* Give you a sensible way to install an OS remotely
+
+When it boots an installed system, it'll actually do a kexec and reboot into the
+actual OS kernel. Overall, its a pretty easy to use and simple way to remotely
```
Comment (jordane) [inline content/osuosl/leveling-up-with-power8.rst]: You should restructure the last sentence here. Maybe something like:

```
Overall, it is an easy and simple way to remotely manage a machine.
```

File: content/osuosl/leveling-up-with-power8.rst [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+manage a machine.
+
+Operating System Setup
+----------------------
+
+The next major challenge was creating a stable operating system environment.
+When I first starting to test these machines, I was using a beta build of RHEL 7
+for ppc64le which was a bit buggy for the architecture. Thankfully 7.1 was
```
Comment (jordane) [inline content/osuosl/leveling-up-with-power8.rst]: The end of the sentence `which was a bit buggy for the architecture` is awkward. Maybe reword to something like:

```
When I first started to test these machines, I was using a bet build of RHEL 7 for ppc64le that contained bugs
```

File: content/osuosl/leveling-up-with-power8.rst [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+
+When it boots an installed system, it'll actually do a kexec and reboot into the
+actual OS kernel. Overall, its a pretty easy to use and simple way to remotely
+manage a machine.
+
+Operating System Setup
+----------------------
+
+The next major challenge was creating a stable operating system environment.
+When I first starting to test these machines, I was using a beta build of RHEL 7
```
Comment (jordane) [inline content/osuosl/leveling-up-with-power8.rst]: `started` instead of `starting`

File: content/osuosl/leveling-up-with-power8.rst [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+for ppc64le which was a bit buggy for the architecture. Thankfully 7.1 was
+released recently which provided a much more stable installer and platform in
+general. Getting the OS installed was the easy part, the more challenging part
+was getting our normal base system up with Chef. This required manually building
```
Comment (jordane) [inline content/osuosl/leveling-up-with-power8.rst]: You state here that the most challenging part was getting the base system up with Chef, but then state that the task that made it difficult (building a client for ppc64le) was actually pretty easy.

File: content/osuosl/leveling-up-with-power8.rst [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+there is no ppc64le builds (yet) for the architecture. So I needed up having to
+build much of the dependencies using mock and hosting it in a local
+repository. Thankfully we didn't require many dependencies, and all of the
+builds had no compile problems.
+
+Storage layout and configuration
+--------------------------------
+
+One of the other interesting parts of this was the storage configuration for the
+servers. These machines came with five 387GB SSDs and ten 1.2TB of SAS disks.
```
Comment (jordane) [inline content/osuosl/leveling-up-with-power8.rst]: the `of` can be dropped completely.

File: content/osuosl/leveling-up-with-power8.rst [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+released recently which provided a much more stable installer and platform in
+general. Getting the OS installed was the easy part, the more challenging part
+was getting our normal base system up with Chef. This required manually building
+a chef client for ppc64le since none existed yet. This was fairly easy to do
+since they have a tool called Omnibus to build chef.
+
+The next challenge was installing all of the base packages and packages we
+needed for running our FTP mirrors. Most of those packages are in EPEL however
+there is no ppc64le builds (yet) for the architecture. So I needed up having to
+build much of the dependencies using mock and hosting it in a local
```
Comment (jordane) [inline content/osuosl/leveling-up-with-power8.rst]: replace `much` with `many`
